src/collection-browser.ts

```diff
@@ -1048,7 +1048,7 @@ export class CollectionBrowser
 
     return html` <tile-dispatcher
       .baseNavigationUrl=${this.baseNavigationUrl}
-      .baseImageUrl=${this.baseImageUrl}
+      .baseImageUrl=${this.baseImageUrl ?? 'https://archive.org'}
```

Comment:
Set this where the actual property is defined instead of where it's being passed in